Source/Core/Common/x64Emitter.h

```diff
@@ -378,19 +380,24 @@ class XEmitter
 
 public:
   XEmitter() = default;
-  explicit XEmitter(u8* code_ptr) : code{code_ptr} {}
+  explicit XEmitter(u8* code_ptr, u8* code_end_ptr) : code(code_ptr), code_end(code_end_ptr) {}
   virtual ~XEmitter() = default;
-  void SetCodePtr(u8* ptr);
+  void SetCodePtr(u8* ptr, u8* end, bool failed = false);
```

Comment:
```suggestion
  void SetCodePtr(u8* ptr, u8* end, bool has_write_failed = false);
```

So that it's explicit what's being set.